src/Responsive.php

```diff
@@ -125,37 +122,17 @@ public static function getPresets(Asset $asset): array
     protected static function getGlideUrl(Asset $asset, string $preset, string $fit, ?string $format = null): ?string
     {
         if ($preset === 'placeholder') {
-            return Statamic::tag('glide:data_url')->params([
-                'preset' => $preset,
-                'src' => $asset->url(),
-                'format' => $format,
-                'fit' => $fit,
-            ])->fetch();
-        }
-
-        $manipulator = self::getManipulator($asset, $preset, $fit, $format);
-
-        if (is_string($manipulator)) {
-            return null;
+            return route('glide-image.placeholder', [
+                'file' => ltrim($asset->url(), '/'),
+            ]);
         }
 
-        $params = $manipulator->getParams();
-
-        $manipulationCacheKey = 'asset::'.$asset->id().'::'.md5(json_encode($params) ? json_encode($params) : '');
-
-        if ($cachedUrl = Glide::cacheStore()->get($manipulationCacheKey)) {
-            $url = Str::ensureLeft(config('statamic.assets.image_manipulation.route'), '/').'/'.$cachedUrl;
-
-            return URL::encode($url);
-        }
-
-        if (config('queue.default') === 'redis') {
-            GenerateGlideImageJob::dispatch($asset, $preset, $fit, $format);
-        } else {
-            GenerateGlideImageJob::dispatchAfterResponse($asset, $preset, $fit, $format);
-        }
+        $signatureFactory = SignatureFactory::create(config('app.key'));
+        $params = $signatureFactory->addSignature($asset->url(), ['preset' => $preset, 'fit' => $fit, 'format' => '.'.$format]);
 
```

Comment:
So because of this signature it will always will be going throught the php?

Reply:
Yes, but its as intended as we have to get the url based on the params.
The signature is also based on the params and is needed so you can't just spam different sizes for an image.